projects/angular-components/package.json

```
@@ -1,10 +1,16 @@
 {
   "name": "@frankframework/angular-components",
-  "version": "1.0.0",
+  "version": "1.0.1",
   "description": "A collection of reusable components designed for use in Frank!Framework projects",
   "main": "",
   "author": "Vivy Booman",
   "license": "Apache-2.0",
+  "homepage": "https://github.com/frankframework/angular-components#readme",
```

The README will already be shown on NPM and there is a link to GitHub, so maybe it would be nice to have the website here.

```suggestion
  "homepage": "https://frankframework.org",
```